cli/compose/convert/service.go

```diff
@@ -357,6 +367,28 @@ func convertFileObject(
 		source = obj.Name
 	}
 
+	// if the config is a Runtime config, that means we don't mount it as a
+	// file in the container. This is used for supporting CredentialSpec
+	// configs.
+	if config.Runtime {
+		if config.Target != "" || config.UID != "" || config.GID != "" || config.Mode != nil {
```

Comment:
Wouldn't target and uid/gid still be configurable? (i.e., mount a config as credential-spec, but specify which uid/gid and/or path to use?

Reply:
No, because a credentialspec is never mounted into the container, or written to the filesystem. And in any case, the actual API type (which is a `ConfigReference` with a `ConfigReferenceRuntimeTarget`) does not allow setting file attributes.

Reply:
🤦‍♂️ ah, yes, makes sense